secretlounge-ng

```diff
@@ -41,7 +41,10 @@ def open_db(config):
 	if type == "json":
 		return JSONDatabase(*args)
 	elif type == "sqlite":
-		return SQLiteDatabase(*args)
+		path = os.path.split(args[0])
+		if not path[0] == '' and not os.path.exists(path[0]):
+			os.makedirs(path[0])
```

Comment:
the exists check can be omitted if you calling makedirs with `exist_ok=True`
also `path[0] != ''` would be nicer

Reply:
Nice suggestions. Added them to the code =)